renderer/src/GuiChar.cpp

```diff
@@ -0,0 +1,95 @@
+#include "GuiChar.hpp"
+#include "stb_image.h"
+
+GuiChar::GuiChar(std::string path)
+{
+	std::vector<float> vertices = {
+			0.5f, 0.5f, 0.0f, 1.0f, 0.0f, //tr
+			0.5f, 0.0f, 0.0f, 1.0f, 1.0f, //br
+			0.0f, 0.0f, 0.0f, 0.0f, 1.0f, //bl
+			0.0f, 0.5f, 0.0f, 0.0f, 0.0f	//tl
```

Comment:
Just something stupid, but could we get the comment indentation consistent here?